opencti-platform/opencti-front/src/utils/outcome_template/engine/apexchartUtils.ts

```diff
@@ -0,0 +1,31 @@
+import { ApexOptions } from 'apexcharts';
+
+const sleep = (delay: number) => new Promise((resolve) => {
+  setTimeout(resolve, delay);
+});
+
+/**
+ * Creates a chart in base64 that can be inject inside a <img /> tag.
+ *
+ * @param chartOptions The ApexChart options to set up the chart.
+ * @returns The base64 data for <img /> tag.
+ */
+const chartDataURI = async (chartOptions: ApexOptions) => {
+  const options = chartOptions;
+  const canvas = document.createElement('canvas');
+  canvas.width = 800;
+  canvas.height = 400;
+  canvas.style.zIndex = '-1';
+  canvas.style.position = 'absolute';
+  document.body.appendChild(canvas);
+  const chart = new ApexCharts(canvas, options);
+  await chart.render();
+  await sleep(1000); // Wait animations are over
```

Comment:
Have you check if there is a way to disable animation in apexchart instead of sleep ?

Reply:
I tried this option but the result was still empty. The sleep option was what was used as accepted answer on a stackoverflow related question